openassessment/xblock/static/sass/oa/utilities/_extends.scss

```diff
@@ -48,16 +48,19 @@
     @include transform(scale(1.0));
   }
 
-  // STATE: is collapsed
-  &.is--collapsed {
-    @include transform(scale(0.99));
-    @include single-box-shadow(0, 1px, 1px, 0, $shadow-l2);
-
+  // don't zoom on hover/focus when step has error
+  &.has--error {
     &:hover, &:focus {
-      @include transform(scale(1.0));
+      @include transform(scale(.99));
```

Comment:
Before, a step would be identified by is--collapsed, now a step is identified by is--showing. The logic just gets flipped here.